### DISCUSSION TITLE
Use UnitAbbreviationsCache.Default to extend a abbreviation

### DISCUSSION THREAD

**Discussion by Matze693** [angularsen/UnitsNet#1415] [Q&A] [answered]
### UseCase
I'm using a multimeter which returns `ElectricResistance` as string, with the pattern `x.xx Ohm`. To parse this with UnitsNet I'm using `UnitAbbreviationsCache.Default.MapUnitToAbbreviation(ElectricResistanceUnit.Ohm, formatProvider, "Ω", "Ohm")`.

### Issue
When using `UnitAbbreviationsCache.Default.MapUnitToAbbreviation` all other abbreviations are overridden. Means, in my case, I'm not able to parse `"4.00 MΩ"` any more.

```csharp
[Fact]
public void ExtendUnitAbbreviationsCacheTest()
{
    const string valueToParse = "4.00 MΩ";
    var expectedQuantity = ElectricResistance.FromMegaohms(4);
    var formatProvider = CultureInfo.InvariantCulture;

    Assert.Equal(ElectricResistance.Parse(valueToParse, formatProvider), expectedQuantity);

    UnitAbbreviationsCache.Default.MapUnitToAbbreviation(ElectricResistanceUnit.Ohm, formatProvider, "Ω", "Ohm");

    // -> The input string '4.00 M' was not in a correct format.
    Assert.Equal(ElectricResistance.Parse(valueToParse, formatProvider), expectedQuantity);
}
```

### Question
Is there any other solution, to extend a unit map with a custom abbreviation, instead of using my own `UnitAbbreviationsCache`?

**Comment by angularsen** [comment, marked as answer]
Late reply, sorry.

What version is this?
Your code works for me on 5.58 and 6.0.0-pre012.

**Reply by angularsen** [reply]
It seems from the comment `The input string '4.00 M' was not in a correct format.` that you maybe accidentally forgot to pass the Ohm character in your testing? `4.00 M` vs `4.00 MΩ`.

**Reply by Matze693** [reply]
Thanks for your reply.

We still use 5.12.0, with 5.59.0 works as expected...
Sorry, my fault to check the version first.

**Reply by angularsen** [reply]
I see, no problem. Glad it worked out!
